superbench/runner/runner.py

```diff
@@ -122,6 +134,17 @@ def __get_mode_command(self, benchmark_name, mode):
                     'superbench.benchmarks.{name}.parameters.distributed_backend=nccl'
                 ).format(name=benchmark_name),
             )
+        elif mode.name == 'mpi':
+            mode_command = (
+                'mpirun '
+                '--allow-run-as-root -hostfile hostfile -map-by ppr:{proc_num}:numa '
+                '{mca_list} {env_list} {command}'
+            ).format(
+                proc_num=mode.proc_num,
+                mca_list=' '.join(f'-mca {k} {v}' for k, v in mode.mca.items()),
+                env_list=' '.join(f'-x {k}={v}' if v else f'-x {k}' for k, v in mode.env.items()),
+                command=exec_command,
+            )
```

Comment:
If mode_command doesn't belong to any support feature, it will return `exec_command`. Is it our design?

Reply:
it will return original command